includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -125,10 +125,22 @@ public function get_settings() {
 					'desc_tip'    => true,
 				),
 
+				array(
+					'title'       => __( 'Header image - Dark Theme', 'woocommerce' ),
+					'desc'        => __( 'URL to an image you want to show in the email header when the email client is using the dark theme/dark mode. Upload images using the media uploader (Admin > Media).', 'woocommerce' ),
+					'id'          => 'woocommerce_email_header_image_dk',
+					'type'        => 'text',
+					'css'         => 'min-width:400px;',
+					'placeholder' => __( 'N/A', 'woocommerce' ),
+					'default'     => '',
+					'autoload'    => false,
+					'desc_tip'    => true,
+				),
+
 				array(
 					'title'       => __( 'Footer text', 'woocommerce' ),
 					/* translators: %s: Available placeholders for use */
-					'desc'        => __( 'The text to appear in the footer of all WooCommerce emails.', 'woocommerce' ) . ' ' . sprintf( __( 'Available placeholders: %s', 'woocommerce' ), '{site_title} {site_url}' ),
+					'desc'        => __( 'The text to appear in the footer of all WooCommerce emails.', 'woocommerce' ) . ' ' . sprintf( __( 'Available placeholders: %s', 'woocommerce' ), '{site_title} {site_address}' ),
```

Comment:
It introduces a regression that doesn't sounds related to this PR.

Reply:
Looks like this was changed in #25630 which was included in the 4.2.0 release that was published after this pull request was submitted. I will update and resubmit.

Reply:
Fixed on this PR.